app/src/main/java/com/wednesday/banklocator/adapter/DatabaseAdapter.kt

```diff
@@ -0,0 +1,61 @@
+package com.wednesday.banklocator.adapter
+
+import android.view.Gravity
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.recyclerview.widget.RecyclerView
+import com.wednesday.banklocator.R
+import com.wednesday.banklocator.model.IfscResponse
+import com.wednesday.banklocator.viewmodel.BankViewModel
+import kotlinx.android.synthetic.main.partial_banks_list.view.bankAddress_tv
+import kotlinx.android.synthetic.main.partial_banks_list.view.bankName_tv
+import kotlinx.android.synthetic.main.partial_saved_banks_list.view.*
+
+class DatabaseAdapter(var item: List<IfscResponse>, private val viewModel: BankViewModel) :
```

Comment:
Please rename this class. 
Also, the data sources should not be items. Always make it plural if it's a list and name it such that it's readable.